sdk/CONTRIBUTING.md

```diff
@@ -1,12 +1,70 @@
 ## Contributing to the `kfp` SDK
-KFP v2 for SDK is under development on the `master` branch.
 
-To contribute to v1, please go to [sdk/release-1.8 branch](https://github.com/kubeflow/pipelines/tree/sdk/release-1.8) to submit your change.
+For developing KFP v2 SDK, use the `master` branch.
+
+For developing KFP v1 SDK, use the [sdk/release-1.8 branch](https://github.com/kubeflow/pipelines/tree/sdk/release-1.8).
 
 For general contribution guidelines including pull request conventions, see [pipelines/CONTRIBUTING.md](https://github.com/kubeflow/pipelines/blob/master/CONTRIBUTING.md).
 
-### Requirements
-All development requirement versions are pinned in [requirements-dev.txt](https://github.com/kubeflow/pipelines/blob/master/sdk/python/requirements-dev.txt).
+### Pre-requisites 
+
+Clone the repo: 
+
+```bash
+git clone https://github.com/kubeflow/pipelines.git && cd pipelines
+```
+
+We suggest using a tool like [virtual env](https://docs.python.org/3/library/venv.html) or something similar for isolating 
+your environment and/or packages for you development environment. For this setup we'll stick with `virtual env`. 
```

Comment:
what's the significance of "virtual env" being in the backticks?

Reply:
removed

Reply:
resolved